spring-webflux/src/main/java/org/springframework/web/reactive/config/ViewResolverRegistry.java

```diff
@@ -182,5 +182,4 @@ public ScriptRegistration() {
 			getViewResolver();
 		}
 	}
-
```

Comment:
For future reference, please do not delete lines such as this one, especially in files that you otherwise have not modified.